cmd/kpod/attach.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	attachFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "detach-keys",
+			Usage: "Override the key sequence for detaching a container. Format is a single character [a-Z] or ctrl-<value> where <value> is one of: a-z, @, ^, [, , or _.",
+		},
+		cli.BoolFlag{
+			Name:  "--no-stdin",
```

Comment:
I think you don't want the "--" at the start here.

Reply:
fixed